src/components/_global/BalTable/BalTableRow.vue

```diff
@@ -37,6 +37,14 @@ function getHorizontalStickyClass(index: number) {
   }
   return '';
 }
+
+function handleColumnClick(column: ColumnDefinition, event: Event) {
+  // Disable default onClick when column is a checkbox
```

Comment:
Clicking a pool row in the voting list goes to the pool page, so we need this code to prevent that behaviour when clicking the checkbox inside the row.